docs/api/pincer.objects.rst

```diff
@@ -0,0 +1,507 @@
+.. currentmodule:: pincer
+
+Pincer Objects Module
```

Comment:
mmmh these would all be the same size titles, `=` `-` and `~` are there to separate it